download.py

```diff
@@ -76,27 +91,45 @@ def download_run_ftp(accession, term, out):
     bool
         True if run was correctly downloaded, otherwise- False
     """
-    url = f"https://www.ebi.ac.uk/ena/portal/api/filereport?accession={accession}&result=read_run&fields=fastq_ftp,fastq_md5&format=json"
+    url = (
+        f"https://www.ebi.ac.uk/ena/portal/api/filereport?accession={accession}"
+        "&result=read_run&fields=fastq_ftp,fastq_md5&format=json"
+    )
     response = requests.get(url)
```

Comment:
I know it can be an overkill, but don't you want to use Pydantic for validating the response and getting what you need from it? That would be cleaner in my opinion. But in a separate PR

Reply:
Yeah, it's a massive overkill. There're just two fields.